packages/tokens-builder/transforms/attribute/scss-value.js

```diff
@@ -0,0 +1,12 @@
+module.exports = (StyleDictionary) => {
+    StyleDictionary.registerTransform({
+        name: 'kbq-scss/value',
+        type: 'value',
+        transformer: (token) => {
+            if (typeof token.value === 'string' && !token.value) {
```

Comment:
если это проверка на пустую строку, мб так и написать?
```suggestion
            if (token.value === '') {
```

Reply:
еще варик заменить null на unset, чтобы не применять функцию трансформации